spec/models/miq_widget_spec.rb

```diff
@@ -763,7 +763,7 @@ def add_dashboard_for_user(db_name, userid, group)
                                      :userid   => @user1.userid,
                                      :group_id => @group2.id
                                     )
-        widget.make_memberof(@ws3)
+        @ws3.add_member(widget)
```

Comment:
still like the idea of `@ws3.members.add(widget)` but unsure if we can get there